docs/administrator-documentation/moderne-platform/how-to-guides/agent-configuration/configure-agent-files-service.md

```diff
@@ -0,0 +1,121 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Configure the agent with file sources
+
+In order for Moderne to obtain information about your organizational structure, you will need to create some configuration files and direct the Moderne agent to said files. This guide will explain how to do both of those things.
+
+## repos.csv
+
+This is a required file, which outlines the repositories and the organizations which they are associated with.
+When provided your organization details, they must be provided in a CSV file with the following format:
+
+| cloneUrl      | branch   | org1    | org2        | org3 |
+|---------------|----------|---------|-------------|------|
+| `https://github.com/openrewrite/rewrite-recipe-bom` | main | Open Source | ALL | |
+| `https://github.com/Netflix/spectator-go` | main | Netflix | Open Source | ALL |
+
+The organizations under `org1`, `org2`, `org3`, etc. represent the hierarchy of organizations. There is no limit to the number of organizations that can be provided via this CSV.
+
+The above example would be used in Moderne DX to generate an organizational listing of the following:
+
+* ALL
+    * Open Source
+        * `https://github.com/openrewrite/rewrite-recipe-bom:main`
+        * Netflix
+            * `https://github.com/Netflix/spectator-go:main`
+
+To generate this `repos.csv`, we recommend using "[repo fetchers](https://github.com/moderneinc/repository-fetchers)".
+
+## commitOptions.txt
+
+This is an optional file, which allows for custom commit options for individual repositories. When no commit options are provided for a specific repositories we will fall back the default commit options which are specified in the [agent configuration](agent-variables.md#all-agent-configuration-variables) or all commit options if no default are provided.
+If you want to provide specific commit options for specific repositories, these can be provided in a text file with the following format:
+```text
+My Organzation ID 1=Branch,PullRequest
+My Organzation ID 2=Branch,Direct,PullRequest
+```
+
+## idMapping.txt
+
+This is an optional file, which set an organization name. By default, an organization name is the same as its ID, these organization IDs are provided in the repos.csv.
```

Comment:
I'm still a bit confused by this >_< sorry.

I think it would help having a real-example here. Like if `repos.csv` has this line:

```
| `https://github.com/openrewrite/rewrite-recipe-bom` | main | Open Source | ALL | |
```

Does that mean there are three organizations created: main, Open Source, and ALL? And the IDs for those organizations exactly match the capitalization and spacing of that?

Reply:
I adjusted the example in the docs. I think this will flow nicely but let me know if it doesn't make sense :)